app/processing/statement/get-statement.js

```diff
@@ -0,0 +1,26 @@
+const { getDetails, getAddress, getDetailedFunding, getScheme, getDetailedPayments } = require('./components')
+const getCalculation = require('../calculation')
+const getPaymentRequest = require('../payment-request')
+const getSettlement = require('../settlement')
+
+const getStatement = async (settlementId) => {

```

Comment:
Something that is unanimous to all of these sub functions is they need to return each value as a string -- well worth a test case